api/src/main/java/me/lokka30/treasury/api/economy/account/Account.java

```diff
@@ -152,4 +154,13 @@ public void fail(@NotNull EconomyException exception) {
      */
     void deleteAccount(@NotNull EconomySubscriber<Boolean> subscription);
 
+    /**
+     * Request the {@link AccountTransaction} history of this {@code Account}
+     *
+     * @param subscription the {@link EconomySubscriber} accepting the transaction history
+     * @author MrIvanPlays
+     * @since {@link me.lokka30.treasury.api.economy.misc.EconomyAPIVersion#v1_0 v1.0}
+     */
+    void retrieveTransactionHistory(@NotNull EconomySubscriber<Collection<? extends AccountTransaction>> subscription);
```

Comment:
What maximum size should the transaction history be?

Or, should this be left in the hands of the economy provider?

e.g. only supply the last 30 transactions.

Reply:
I will add a parameter, int lastTransactionsCount

Reply:
I am unsure whether 'request all transactions':
* should be possible
  a. should be done with this method, e.g. maybe `lastTransactionsCount = -1`
  b. should be done with another method
* should not be possible

Reply:
I documented it is possible